packages/frontend/src/utils/project/getContractSection.ts

```diff
@@ -23,6 +28,23 @@ export function getContractSection(
     return makeTechnologyContract(contract, project, isUnverified)
   })
 
+  const escrows = project.config.escrows
+    .sort(moreTokensFirst)
+    .map((escrow) => {
```

Comment:
maybe a better pattern would be to filter out the hidden and not newFormat after the sort, because currently it is a bit hard to understand at first glance